Tasks/CargoAuthenticateV0/package.json

```diff
@@ -24,12 +24,12 @@
     },
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
-        "@types/node": "^10.17.0",
+        "@types/node": "^16.11.39",
         "@types/mocha": "^5.2.7",
         "@types/uuid": "^8.3.0",
         "azure-pipelines-tasks-artifacts-common": "1.0.1",
-        "azure-pipelines-tasks-packaging-common": "3.206.0",
-        "azure-pipelines-task-lib": "^3.1.7",
+        "azure-pipelines-tasks-packaging-common": "3.214.0",
+        "azure-pipelines-task-lib": "^4.0.1-preview",
```

Comment:
sorry, didn't notice last time, could you please bump the version to 4.1.0 according to [the update guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode16.md#upgrading-tasks-to-node-16)?